data-reconciliation-app/src/utils/api-result.ts

```diff
@@ -0,0 +1,16 @@
+import { ServiceResult } from "./service-result";
+
+export interface CCFResult {
+  statusCode: number;
+  body: any;
+}
+
+export class ApiResult {
+  public static Response<T>(result: ServiceResult<T>): CCFResult {
```

Comment:
Can we organise this to `ApiResult.Success`, `ApiResult.Failure`, `ApiResult.AuthFailure` to provide better context for the code reader?